### PR TITLE
Set default value for spread type

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/trade-params/FullTradeParams.js [hunk 1]
```diff
@@ -399,7 +399,7 @@ export default class FullTradeParams extends Component {
                 }
                 {showDigitBarrier &&
                     <DigitBarrierCard
-                        barrier={trade.barrier}
+                        barrier={+(trade.barrier)}
```
Comment (borisyankov) [inline src/trade-params/FullTradeParams.js]: No need for ()

File: src/payout-picker/StakeCard.js [hunk 2]
```diff
@@ -22,12 +22,14 @@ export default class StakeCard extends Component {
 
     stepUp() {
         const { amount, onAmountChange } = this.props;
-        onAmountChange({ target: { value: amount + step } });
+        const newAmount = +(amount) + step;
```
Comment (borisyankov) [inline src/payout-picker/StakeCard.js]: No need for ()

File: src/payout-picker/StakeCard.js [hunk 3]
```diff
@@ -22,12 +22,14 @@ export default class StakeCard extends Component {
...
+        onAmountChange({ target: { value: newAmount.toString() } });
     }
 
     stepDown() {
         const { amount, onAmountChange } = this.props;
-        onAmountChange({ target: { value: amount - step } });
+        const newAmount = +(amount) - step;
+        onAmountChange({ target: { value: newAmount } });
```
Comment (borisyankov) [inline src/payout-picker/StakeCard.js]: Why no toString( ) here?

Comment (qingweibinary) [inline src/payout-picker/StakeCard.js]: Because it's already guarantee to be string from the caller ~

**Pull request merged**